docs/core/tools/dotnet-new-update.md

```diff
@@ -9,49 +9,81 @@ ms.date: 04/29/2021
 
 ## Name
 
-`dotnet new --update-check` - Checks for available updates for installed template packages.
-
-`dotnet new --update-apply` - Applies updates to installed template packages.
+`dotnet new update` - updates installed template packages.
+`dotnet new update --check-only` - Checks for available updates for installed template packages without applying them.
```

Comment:
Extra line needed here or extra spaces at the end of the first line to force the line break (https://daringfireball.net/projects/markdown/syntax#p)
Otherwise those two examples appear in a single paragraph.

Reply:
I decided to remove the second line - it's not needed anymore. It is also explained in `Description` and doesn't look similar to other articles.